amity/person.py

```diff
@@ -5,17 +5,65 @@ class Person(object):
         in the Amity System. It relies heavily on the information it
         gets from it's Child Classes.
     '''
+    number_of_fellows = 0
+    number_of_staff = 0
+    total_persons = 0
+    persons = {"Fellows": {}, "Staff": {}}
+    error = ""
+
     def __init__(self, uuid, name, role):
+        if role == "Fellow":
+            role_key = role + "s"
+        else:
+            role_key = role
+        self.persons[role_key][uuid] = {}
+        self.persons[role_key][uuid]["uuid"] = uuid
```

Comment:
Instead of using `uuid` it will be better if you used something like `fellow_id` or `staff_id`. uuid is not a very descriptive variable name.